### PR TITLE
tooltip with prime

### PR DESCRIPTION
**Pull request by tweetgeek**

### CODE REVIEW

File: lib/js/components/Tooltip/Tooltip.stories.ts [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+	},
+	template: `
+    <div style="padding: 60px; width: 100%;display: flex;  justify-content: center">
+      <div style="padding: 60px;">
+        <span>This is a text with </span>
+        <tooltip
+          :text="text"
+          :is-disabled="isDisabled"
+          :placement="placement"
+          :isPointerVisible="isPointerVisible">
```
Comment (pioziol123) [inline lib/js/components/Tooltip/Tooltip.stories.ts:26]:
```suggestion
          :is-pointer-visible="isPointerVisible">
```

Comment (pioziol123) [reply lib/js/components/Tooltip/Tooltip.stories.ts:26]: and other places

File: lib/js/components/Tooltip/Tooltip.stories.ts [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+		return { ...args };
+	},
+	template: `
+    <div style="padding: 60px; width: 100%;display: flex;  justify-content: center">
+      <div style="padding: 60px;">
+        <tooltip
+          :text="text"
+          :is-disabled="isDisabled"
+          :placement="placement"
+          :isPointerVisible="isPointerVisible">
```
Comment (pioziol123) [inline lib/js/components/Tooltip/Tooltip.stories.ts:72]:
```suggestion
          :is-pointer-visible="isPointerVisible">
```

File: lib/js/components/Tooltip/Tooltip.stories.ts [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+		return { ...args };
+	},
+	template: `
+    <div style="padding: 60px; width: 100%;display: flex;  justify-content: center">
+      <div style="padding: 60px;">
+      <tooltip
+        :text="text"
+        :is-disabled="isDisabled"
+        :placement="placement"
+        :isPointerVisible="isPointerVisible">
```
Comment (pioziol123) [inline lib/js/components/Tooltip/Tooltip.stories.ts:101]:
```suggestion
        :is-pointer-visible="isPointerVisible">
```

File: lib/js/components/Tooltip/Tooltip.vue
Comment (pioziol123) [inline lib/js/components/Tooltip/Tooltip.vue:1]: Zrzut ekranu 2024-06-21 o 10 24 53
Tooltip slips over the button. It happens with top/bottom but works well with left/right.
Also I cannot use interactive mode. I gets black canvas. I dunno if its me or it is something with configuration

Comment (pioziol123) [reply lib/js/components/Tooltip/Tooltip.vue:1]: Zrzut ekranu 2024-06-21 o 10 36 14
It works fine when disabling and then enabling arrow

File: lib/js/primevue.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import PrimeVue from 'primevue/config';
+import Aura from '@primevue/themes/aura';
+import Tooltip from 'primevue/tooltip';
+
+export const initializePrimeVue = (app) => {
+	app.use(PrimeVue, {
+		theme: {
+			preset: Aura,
+			options: {
+				darkModeSelector: '', // enabling dark mode in storybook makes all components canvas black
```
Comment (tweetgeek) [inline lib/js/primevue.ts:10]:
```suggestion
				darkModeSelector: '.app-dark', // enabling dark mode in storybook makes all components canvas black
```

File: lib/js/components/Tooltip/Tooltip.vue [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+	</span>
+	<div v-else v-pv-tooltip:[tooltipParams]="options" tabindex="-1">
+		<slot />
+	</div>
+</template>
+
+<style lang="scss">
+@import '../../../styles/settings/typography/tokens';
+
+.ds-tooltip-text {
```
Comment (pioziol123) [inline lib/js/components/Tooltip/Tooltip.vue:13]: where those classes are used?

Comment (pioziol123) [reply lib/js/components/Tooltip/Tooltip.vue:13]: Ah. I see now

**Review by pioziol123 — APPROVED** [review]

**Pull request merged**